widgetssdk/src/main/java/com/glia/widgets/view/dialog/base/DialogDelegate.kt

```diff
@@ -4,23 +4,19 @@ import androidx.appcompat.app.AlertDialog
 import com.glia.widgets.core.dialog.model.DialogState
 
 internal interface DialogDelegate {
-    fun showDialogIfNoDialogPresent(showDialogCallback: () -> AlertDialog)
-    fun forceShowDialog(showDialogCallback: () -> AlertDialog)
+    fun showDialog(showDialogCallback: () -> AlertDialog)
     fun updateDialogState(dialogState: DialogState): Boolean
-    fun dismissAlertDialog()
+
+    //    fun dismissAlertDialog()
+//    fun resetDialogState()
```

Comment:
We can probably remove this, can't we?

Reply:
Yes, will do it after manual testing, to be sure that we don't need them at all.